runtime/shared_common/ScopeManagerImpl.cpp

```diff
@@ -96,10 +96,6 @@ SH_ScopeManagerImpl::scHashEqualFn(void* left, void* right, void *userData)
 		Trc_SHR_SMI_scHashEqualFn_Exit2();
 		return 0;
 	}
```

Comment:
The length comparison here is redundant and can be removed (that test is included in `J9UTF8_EQUALS()`).